src/storable/tuples.rs

```diff
@@ -11,26 +11,22 @@ where
     fn to_bytes(&self) -> Cow<[u8]> {
         match Self::BOUND {
             Bound::Bounded { max_size, .. } => {
-                let mut bytes = vec![0; max_size as usize];
                 let a_bytes = self.0.to_bytes();
                 let b_bytes = self.1.to_bytes();
-
                 let a_bounds = bounds::<A>();
                 let b_bounds = bounds::<B>();
-
-                let a_max_size = a_bounds.max_size as usize;
-                let b_max_size = b_bounds.max_size as usize;
-
-                debug_assert!(a_bytes.len() <= a_max_size);
-                debug_assert!(b_bytes.len() <= b_max_size);
-
-                bytes[0..a_bytes.len()].copy_from_slice(a_bytes.borrow());
-                bytes[a_max_size..a_max_size + b_bytes.len()].copy_from_slice(b_bytes.borrow());
-
+                let a_max = a_bounds.max_size as usize;
```

Comment:
nit: I'd keep it `a_max_size`, as `a_max` sounds like the maximum value of `a`.

Reply:
done.